WMF Framework/NavigationBar.swift

```diff
@@ -432,7 +432,11 @@ public class NavigationBar: SetupView, FakeProgressReceiving, FakeProgressDelega
     var extendedViewHeight: CGFloat {
         return extendedView.frame.size.height
     }
-    
+
+    @objc var backgroundViewY: CGFloat {
```

Comment:
Could this be named `hiddenHeight` for consistency with `visibleHeight`? `backgroundView` is an implementation detail and private, so having this named `backgroundViewY` could be confusing. `hiddenHeight` could be set the same way as `visibleHeight` in `layoutSubviews`, equal to `barTransformHeight + extendedViewTransformHeight + underBarTransformHeight`. `totalTransform` is then `0 - hiddenHeight` and `self.navigationBar.hiddenHeight` is added to the calculation in `WebViewController`

Reply:
Sounds good 👍 Will update.

Reply:
it should be positive - `hiddenHeight` == the total height that is hidden, similar to how `visibleHeight` == the total height that is visible.  It should be `barTransformHeight + extendedViewTransformHeight + underBarTransformHeight` and the transform is negative, `0 - hiddenHeight`

Reply:
Oops I think I should have had coffee before reading your initial comment ;) Will remedy that now hehe.

Reply:
Fixed!